.bazelrc

```diff
@@ -53,6 +53,9 @@ build --per_file_copt=external/com_google_tcmalloc@-DANNOTATE_MEMORY_IS_INITIALI
 build --per_file_copt=external/com_google_protobuf@-Wno-deprecated-this-capture
 build --host_per_file_copt=external/com_google_protobuf@-Wno-deprecated-this-capture
 
+## simdutf
+build --per_file_copt=external/simdutf@-Wno-unused-const-variable
```

Comment:
Do we need this? I though we didn't build with WError on workerd?

Reply:
We do build with `-Werror`, but only in CI.